webui/react/src/pages/F_ExpList/glide-table/GlideTable.tsx

```diff
@@ -475,7 +475,22 @@ export const GlideTable: React.FC<GlideTableProps> = ({
         _: () => loadingCell,
         Loaded: (rowData) => {
           const columnId = columnIds[col];
-          return columnDefs[columnId]?.renderer?.(rowData, row) || loadingCell;
+          let cell = columnDefs[columnId]?.renderer?.(rowData, row);
+          switch (cell.kind) {
+            case GridCellKind.Text:
+            case GridCellKind.Number:
+              if (!cell.displayData || cell.displayData === '') {
+                cell = {
+                  ...cell,
+                  displayData: '-',
+                  themeOverride: { ...cell.themeOverride, textDark: 'LightGrey' },
```

Comment:
Not really sure why, but using variable color such as `--theme-surface-on-weak` leads to weird inconsistent behavior

Reply:
could you try using `getCssVar` here to get the resolved variable value?

Reply:
That works! Thanks!
Also the `--theme-surface-on-weak` is just the default text color for the table